Source/Wit/Public/Wit/Utilities/WitHelperUtilities.cpp

```diff
@@ -349,6 +350,12 @@ bool FWitHelperUtilities::SaveClipToAssetFile(const FString& ClipDirectory, cons
 	FMemory::Memcpy(CacheAsset->ClipData.GetData(), ClipData.GetData(), ClipData.Num());
 			
 	(void)CacheAsset->MarkPackageDirty();
```

Comment:
I wonder is it necessary to mark both the asset and the package as dirty? Maybe only the package is needed?

Reply:
I found making the asset dirty doesn't help on saving.